sql/core/src/main/resources/org/apache/spark/sql/execution/ui/static/spark-sql-viz.css

```diff
@@ -33,7 +33,7 @@
 }
 
 /* Highlight the SparkPlan node name */
-#plan-viz-graph svg text :first-child {
+#plan-viz-graph svg text :first-child, #plan-viz-graph svg text :nth-child(2) {
```

Comment:
Is there any side-effect with `:nth-child(2)`?

Reply:
I  concerned about side effects but another idea is a little bit tricky.
I'll submit another one.